client/src/Pages/User/Vendors/VendorList.js

```diff
@@ -26,38 +35,149 @@ const GET_VENDORS_QUERY = gql`
+                <li>Snacks</li>
+                <li>Coffee</li>
+            </ul>
+        </div> */}
+        </Fragment>
+    );
+};
 
-    const handleClick = () => {
-        // Go to this particular vendor's detail page
-        return history.push(`/user/vendors/${vendor.slug}`);
+class HeaderExclusion extends React.Component {
+    screenOptions = {
+        headerShown: false
     }
+}
 
-    return (
-        <div style={{ backgroundImage: `url(${vendor.imageURL})` }} className="vendorcard" onClick={handleClick}>
-            <h1>{vendor.name}</h1>
-        </div>
+const BottomAppBar = ({ }) => {
+    return(
+            <AppBar position="sticky" color="white">
+            <BottomNavigation className="stickToBottom">
+            <Grid container>
+                <Toolbar className="bottomBar">
+                <div>
+                    <FontAwesomeIcon className="barIconCart" icon={faShoppingCart} flexItem/>
+                    <p class="iconText">Cart</p>
+                </div>
+                    <Divider orientation="vertical" flexItem />
+                <div>
+                    <FontAwesomeIcon className="barIconReceipt" icon={faReceipt} flexItem/>
+                    <p class="iconText">Orders</p>
+                </div>
+                </Toolbar>
+            </Grid>
+            </BottomNavigation>
+            </AppBar>
     )
 }
 
-const VendorList = ({ }) => {
+const VendorList = ({ classes }) => {
     const { data, loading, error } = useQuery(GET_VENDORS_QUERY);
 
     if (error) return <p>Error...</p>;
     if (loading) return <p>Loading...</p>;
     if (!data) return <p>No data...</p>;
 
-    const vendors = data.vendorMany;
+    const vendors = [
+        {
+            name: "East West Tea",
+            hours: "7:00pm - 10:00pm",
+            keywords: ["Boba Tea", "Snacks", "Vegan / Non-Dairy"],
+            image:
+                "https://images.squarespace-cdn.com/content/v1/58559451725e25a3d8206027/1486710333727-C3OXR7PA5G2YGEZHNK6R/ke17ZwdGBToddI8pDm48kCHChmuivJZ1Va5ov3ZJeg17gQa3H78H3Y0txjaiv_0fDoOvxcdMmMKkDsyUqMSsMWxHk725yiiHCCLfrh8O1z4YTzHvnKhyp6Da-NYroOW3ZGjoBKy3azqku80C789l0ouw-8l5B_J38LMU7OZFvYcSGirBhY_3j1yQtntvGS73bypqQ-qjSV5umPUlGbQFAw/DSC_0064.jpg?format=1500w",
+            closed: false,
+        },
+        {
+            name: "Coffeehouse",
+            hours: "7:30am - 5:00pm",
+            keywords: ["Coffee", "Snacks", "Decaf"],
+            image:
+                "https://lh5.googleusercontent.com/p/AF1QipNZ0aMBlyVAc6GXs1dVZTMOu6rn0I9G1ZBVJPJz",
+            closed: true,
+        },
+        {
+            name: "Grillosophy",
+            hours: "7:00pm - 10:00pm",
+            keywords: ["Hamburgers", "Sandwiches", "Milkshakes", "Agua Fresca"],
+            image:
+                "https://images.unsplash.com/photo-1522244451342-a41bf8a13d73?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1650&q=80",
+            closed: false,
+        },
+    ];
```

Comment:
If we're using hardcoded data, there's no need to do the query. I'd add a boolean parameter to the `VendorList` component that determines whether or not data fetching is done, and use either the result of that or the hardcoded data accordingly.